### PR TITLE
Bootstrap 3 Migration: CommCare Supply Setup > Advanced Settings

### PR DESCRIPTION
**Pull request by pnaw94**
scr01
@biyeun

### CODE REVIEW

File: corehq/apps/commtrack/forms.py [hunk 1]
```diff
@@ -76,15 +80,15 @@ def __init__(self, *args, **kwargs):
...
                 'consumption_optimal_window',
-                'individual_consumption_defaults',
+                PrependedText('individual_consumption_defaults', ''),
             ),
             Fieldset(
                 _('Phone Settings'),
-                'sync_consumption_fixtures',
+                PrependedText('sync_consumption_fixtures', ''),
             ),
             ButtonHolder(
```
Comment (biyeun) [inline corehq/apps/commtrack/forms.py]: see the `FormActions` part in https://www.commcarehq.org/styleguide/forms/#b3-migration

**Comment by pnaw94** [comment]
@biyeun Updated

**Comment by biyeun** [comment]
@pnaw94 conflicts

**Comment by pnaw94** [comment]
@biyeun updated

**Pull request merged**